docs/readme-cmake.md

```diff
@@ -1,88 +1,59 @@
-
-Download Visual Studio, and make sure to install the following features: 
-- "Desktop Development with C++", 
-- "C++ Clang Compiler for Windows", 
-- "C++ clang-cl for vXXX build tools", and 
-- "C++ CMake tools for Windows".
-
-Please note that it is recommened to use VS 2022 (or higher) for its better CMake integration.
-
-### Windows Build Process
-
-1. Open the endless-sky folder that you cloned in the initial step.
-2. Wait while Visual Studio loads everything. This may take a few minutes the first time, but should be relatively fast on subsequent loads.
-3. On your toolbar there should be a pulldown menu that says "debug" or "release." Select the version you want to build.
-4. Hit the "build" button, or find the "Build" menu and select "Build All"
-5. In the status window it will give a scrolling list of actions being completed. Wait until it states "Build Complete"
-6. At this point you should be able to launch the recently completed build by hitting the F5 key or the run button. The recently build executable and essential libraries can be found in your Endless-Sky folder, nested within a subfolder created by Visual Studio during the build process
-
-#### Notes
+The game's root directory, where your `git clone`d files reside, will be your starting point for compiling the game. You can use `cd endless-sky` to enter the game's directory.
 
-##### Earlier versions of Visual Studio
+Next, you will need to install a couple of dependencies to build the game. There are several different ways to build Endless Sky, depending on your operating system and preference.
 
-If you are on an earlier version of Visual Studio, or would like to use an actual VS solution, you will need to generate the solution manually as follows:
+- [Windows](#windows)
+- [Windows (MinGW)](#windows-mingw)
+- [MacOS](#macos)
+- [Linux](#linux)
 
-```powershell
-> cmake --preset clang-cl -G"Visual Studio 17 2022"
-```
+## Installing build dependencies
 
-This will create a Visual Studio 2022 solution. If you are using an older version of VS, you will need to adjust the version. Now you will find a complete solution in the `build/` folder. Find the solution and open it and you're good to go!
+### Windows
 
-#### Using Microsoft Visual C++
+It is recommended to use the toolchain from Visual Studio to build the game (regardless of the IDE you wish to use).
 
-Using MSVC to compile Endless Sky is not supported. If you try, you will get a ton of warnings (and maybe even a couple of errors) during compilation.
+Download [Visual Studio](https://visualstudio.microsoft.com/downloads/#visual-studio-community-2022) (if you do not want to install Visual Studio, you can alternatively download the [VS Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022)), and make sure to install the following components:
```

Comment:
```suggestion
Download [Visual Studio](https://visualstudio.microsoft.com/downloads/#visual-studio-community-2022) and make sure to install the components listed below. The "Community" edition is appropriate for working on Endless Sky.
If you do not want to install Visual Studio, you can alternatively download just the [VS Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
```
Or something else, but we should probably give some advice on what edition to choose to make it easier for new users.

Reply:
better?